spring-integration-core/src/main/java/org/springframework/integration/dsl/BaseIntegrationFlowDefinition.java

```diff
@@ -2829,6 +2840,26 @@ public B trigger(MessageTriggerAction triggerAction,
 		return handle(new ServiceActivatingHandler(triggerAction, "trigger"), endpointConfigurer);
 	}
 
+	/**
+	 * Adds one or more {@link ChannelInterceptor} implementations
+	 * to the current {@link #currentMessageChannel}, in the given order, after any interceptors already registered.
+	 * @param interceptorArray one or more {@link ChannelInterceptor}s.
+	 * @return the current {@link BaseIntegrationFlowDefinition}.
+	 * @throws IllegalArgumentException if no interceptors or at least one null value are provided
+	 * @since 5.3
+	 */
+	public B interceptor(ChannelInterceptor... interceptorArray) {
```

Comment:
Well, our style to make methods on this builder as verbs, in other words commands: do this or that.
Look at this as EIP-operators!

Therefore if we add this feature here, it has to be named as `intercept()`.